### DISCUSSION TITLE
Group Buy Poll

### DISCUSSION THREAD

**Discussion by stonepreston** [FundamentalFrequency/.github#2] [Polls]
I am trying to gauge interest for those who would like to do a US-based group buy. I am curious if people would also like to get the acrylic case parts in addition to the PCB, since its a bit of a hassle if you dont have access to a laser cutter.

**Comment by BonIverFan** [comment]
Take my money! Amazing job man!

**Reply by ehafh** [reply]
exciting project!  hope I can snag a pcb and case

**Comment by aldaw** [comment]
How can I contact you? :)

**Comment by iidexic** [comment]
As for the case, I'd actually probably prefer just PCB if there weren't square key switches involved, it's tough to cut square holes.

**Comment by raychmond** [comment]
Please count me in, I really like this super awesome project. I already have all the parts except the PCB and the case. I live in Canada though.

**Reply by Dp666me** [reply]
You can make the cases at a local library. Check for the cutters near you

**Reply by raychmond** [reply]
Thank you! I didn't know we can laser cut at a library. I will definitely do some checks.

**Reply by stonepreston** [reply]
If you are near a decently sized city maker spaces are also becoming popular. That’s what I used

**Reply by raychmond** [reply]
If in the end the Canada-based group buy is available or you can ship to Canada, I'd be very happy to order everything from here. Thank you for sharing such an amazing project, it's been something I hope to build but never be able to do so.

**Reply by Dp666me** [reply]
I'm in Canada too

**Reply by baldego1** [reply]
I'm also in Canada, and would buy everything here.

**Comment by mingusdude** [comment]
Man I am so in support of this. I literally just started coding so I could expand from what has been exclusively analog synth builds. You made my morning a few days ago when that first article hit. Anyway. You did a rad thing. Thank you. I really only needed the pcb, but if you make cases I'll buy one too. Please keep me posted!

**Comment by HowdyHolto** [comment]
I’m down for either, and I just wanted to add that I would love to chat with you about some other ways I may be able to support the project. This project hits a big passion cross-section for me and I’m extremely impressed by the work you’ve put in. I also have a “mad design lab” as my friends like to call it and quite the assortment of tools and machines that might be of interest for developing the physical side of the project further. Cheers and again… Amazing work. So excited about this.

**Comment by atardigradestory** [comment]
Am interested but wouldn't want to have to bail if it turned out to be more than I could afford, and I'm not too familiar with estimates on the potential cost of the acryllic case. Could you give a worst-case-scenario estimate for the groupbuy, in case others are in a similar situation?

**Reply by atardigradestory** [reply]
Also my experience with FreeCAD is slim and C++ virtually non-existent but I've been meaning to learn the latter sooner rather than later and am so excited to be able to make contributions to this project where I can find things to do :D

**Reply by stonepreston** [reply]
I cant really give a group buy estimate, as ive not done one before so Im not too sure what kind of volume discount we could get on everything. That being said, for a one-off, non group buy build you are looking at around 250-300 USD for everything but the Pi4. Closer to 250 if you lasercut the case yourself, closer to 300 if not a bit more if you have to pay to have it cut.

**Comment by jantuitman** [comment]
This looks like a very interesting project and I might help with the software… but there is one big obstacle for me: I can’t solder anything myself because my hands are pretty unreliable for fine movements (neurological condition I was born with). So I would love to group buy if I can find someone who wants to assemble my buy for me. And I am willing to pay for that as well. Hoping this will become a kickstarter…

**Reply by atardigradestory** [reply]
I'd be happy to

**Reply by harlowjg** [reply]
Also willing to help.

**Reply by webdevotion** [reply]
Can help. From BE.

**Reply by pixiesue** [reply]
Hello,
You can join us on discord to chat about buying boards and help Jan:

https://discord.gg/WH9WK6gM

Would we need to send him a full build with Pi4 included?
I will have PCBs to spare but no Pi4 sadly...

**Reply by LucidSD** [reply]
Random sorry to add a redundant question to this board but provided I paid for parts+labor would anyone be able to assist in soldering in Los Angeles, CA. 

This is a project that seems really amazing but my last solder project was just a mess (cheap Ebay solder kit so my partially my fault for getting cheap tools I guess + inexperience) and I ruined some cool retro gear. 

Would be willing to pay for labor to avoid having to buy a whole new soldering kit and risk any bridging mistakes (I read  this is a somewhat tricky solder job due to how close the solder points are to each other)

**Comment by companyofquail** [comment]
Nice project. Looking forward to buying a kit and checking it out.

**Comment by kc3lai** [comment]
Very nice project and am interested to buy it as a kit.   I wouldn't mind if you charge some premiums over the parts to support the project. Great work!

**Comment by scowije** [comment]
Interested

**Comment by Dp666me** [comment]
I'm in for sure

**Comment by benpocalypse** [comment]
Regardless of whether or not a group buy occurs, I'm ordering the parts and building one of these for myself no matter what. The more of these we can get into folks hands, the better. If it works at all like it appears, you've made an amazing product here. Something I always wanted to do, just never had the time - so thanks for that!

**Reply by harlowjg** [reply]
I will be doing the same!

**Comment by rafael-azevedo** [comment]
We should consider a group buy for all the parts as well.

**Reply by HowdyHolto** [reply]
Agree 100. With some extra for whoever has the program management brain to pull it off haha.

**Reply by stonepreston** [reply]
Alright yeah definitely open to doing that as well. I’d update the poll but it would reset the votes. If people would rather do that I’m down

**Reply by companyofquail** [reply]
Group buy for parts in USA sounds great

**Comment by nash0rn** [comment]
A Europe-based group buy would be nice, too

**Reply by winfr34k** [reply]
I'd love to take part in that, too ;D

**Reply by pixiesue** [reply]
I made a thread about it, please feel free to contribute there.
https://github.com/orgs/FundamentalFrequency/discussions/6#discussion-4157253

We might want to wait for a maybe soon RGB friendly revision of the PCB?
The case would be the same though.

Anyway we can start to source Pi4 and Teensy4.1 anywhere we can.

We have to choose encoders, key switches and keys as well.
Please share if you have ideas about what would be cool to have!  :)

*here is my stupid first idea*
![image](https://user-images.githubusercontent.com/107801377/174501164-e6b075a0-116b-401c-a8f1-8739b7c7c50d.png)

**Reply by Dp666me** [reply]
Nice I really like that

**Comment by Dp666me** [comment]
If we do the order can we get them to install the parts? I was looking at ordering some tonight and saw that is an option. Is their a BOM file to help with that?

**Reply by stonepreston** [reply]
the issue with that is you have to solder the top plate of the case between the pcb and the switches as shown [here](https://github.com/FundamentalFrequency/LMN-3-Build-Guide#switches), so it doesnt really make it worth it. You could have them solder the diodes, but thats pretty easy to do anyways. Plus thats the majority of the fun :D

If these were SMD parts that would be one thing, but since its all through hole it doesnt really make sense since its pretty easy to do yourself. Im terrible at soldering and didnt have a single issue

**Comment by diriglow** [comment]
Created a GitHub account just to stay up to date with this; it looks awesome, thank you. Coming from a Norns Shield, this seems like a really great project. I am really looking forward to a PCB Group buy, as when I tried ordering with JLCPCB, I got in over my head with the specifics and wasn't sure. Thanks again.

**Reply by pixiesue** [reply]
Same as you, it would be my 1st serious build with a lot of soldering so I created an account here to get help if needed.

I ordered from JLCPCB it was dead simple, you just have to provide the gerbers.zip that you can find here.

https://github.com/FundamentalFrequency/LMN-3-ECAD/releases
![image](https://user-images.githubusercontent.com/107801377/174644828-df8aa3c1-12fc-4a2d-94cd-9eab157306d4.png)

Then you just have to drag it here from your computer and you'll have your quotes for PCBs at specs.
Everything will be populated with the relevant infos. You might chose a different PCB color if you fancy it (I chose white).

![image](https://user-images.githubusercontent.com/107801377/174644751-edd41260-6c99-47a0-be41-27ca8ee03545.png)

I learned that gerber files are the industry standard to share PCB designs, they'll know how to handle it.

**Comment by Dp666me** [comment]
I ordered my board. It was the cheapest part

**Reply by pixiesue** [reply]
Yes, PCBs were maybe the easier part to get for me...   :)

Teensies and Pi4s are harder to source but Hyperpixel screens are easy to find (for now).

Key switches and caps are kind of a pain in the a** to choose.

The exact thumb joystick is only available in the US?

Anyway, I hope to have my unit build in july.

**Reply by raychmond** [reply]
I tried PCBWAY. It costs more than $80CAD shipping to Canada with min 5 boards. I think that's why we wanted to initiate the group buy at the first place. Does JLCPCB require minimum 5 boards, and how much did you end up with? Thank you!

**Reply by stonepreston** [reply]
@raychmond yes it’s 5 min at jlc. And yes that will be the purpose of the group buy

**Reply by raychmond** [reply]
Thank you for confirming @stonepreston

**Reply by stonepreston** [reply]
@raychmond Since you are in Canada, you might not be able to take part in a US-based group buy. You might make a post searching out other canadians so that you guys can order a handful of boards

**Reply by raychmond** [reply]
@stonepreston 
Understood. I will open a new post to share ordering info and laser cutter locations. Thank you!

**Reply by kc3lai** [reply]
@raychmond I'm in Canada as well. Will you be interested to split the PCB cost?

**Reply by raychmond** [reply]
@kc3lai I'm happy to do so. Do you mind if you can discuss the Canada specific topics in the new post at #13? 

Thank you!

**Comment by mingusdude** [comment]
I got a quote from LHD at 50 for 5, and that's with purple solder mask yellow silk screen. Just colors I like... Anyway, the lady there is trying to recruit me for a sales job now that I asked a million questions,  so maybe I can get em cheaper? Regardless 5 for 50 is a good starting point right? @stonepreston let me know and I'll get you the pdf with the quote. This project. It's the proverbial poop. Thank you. I really needed this.

**Reply by stonepreston** [reply]
50 for 5 is about what i paid including shipping to the US

**Comment by mingusdude** [comment]
Yeah that was with shipping of course.

**Comment by raychmond** [comment]
Teensy 4.1 without Ethernet version is in stock now. The batch in July will have the normal version.

**Reply by stonepreston** [reply]
the standard 4.1 teensy is out of stock on the PJRC site, but I found it on adafruit in stock. Probably available elsewhere as well

**Comment by deltronxzero** [comment]
I'm debating buying the parts now or later as I'm just super excited to start this project!! Raspberry pi and Teensy are seemingly the current bottleneck items to obtain??? Is there a specific model you used RAM wise?? I read that new shipments for US won't be until April of 2023 for Pi4. Dragging my feet knowing this 200$ item, cost 45$ originally..

**Reply by raychmond** [reply]
Follow rpilocator on twitter or check his website rpilocator.com regularly (better to follow him, all the stocks run out in a few minutes.). You will have a good chance to buy Pi4.

**Reply by stonepreston** [reply]
@deltronxzero 4 GB RAM and up is recommended, you might be able to get by with 2GB but ive only used 4 GB model in testing

**Reply by stonepreston** [reply]
the pi4 shortage is really unfortunate, but yeah, not much we can do really.

**Reply by stonepreston** [reply]
also, even if you dont want to shell out for a pi, you can still build the PCB/case, and just use it as a dev board to develop and test the DAW running on your computer. Not quite as fun, but still something.

**Reply by deltronxzero** [reply]
It is more than worth it considering I paid what I did for the OG op1 with specs not even close to the pi. I want to build it sooner than later forsure! Do I need teensy with ethernet?? I saw someone they're available now without ethernet.

**Reply by stonepreston** [reply]
@deltronxzero no you dont need one with ethernet, although I think they are only out of stock on the PJRC site. If you look at other vendors such as [adafruit](https://www.adafruit.com/product/4622) you will likely find the standard 4.1 teensy is available

**Reply by deltronxzero** [reply]
@stonepreston thank you! Decided I want to tackle this sooner than later.. hopefully this will be my last question for a while, but would I be ok using linux Mint to install everything?? I saw it said it was "based" on Debian and Ubuntu. Not quite sure if that's what I need. I found a fairly cheap laptop that is using that OS currently.

**Reply by diriglow** [reply]
Heads up Adafruit just posted 100 RPi4 2GB units. Based on the state of things, I’m going to give it a shot when the PCB order happens.

**Reply by Dp666me** [reply]
> Heads up Adafruit just posted 100 RPi4 2GB units. Based on the state of things, I’m going to give it a shot when the PCB order happens.

I don't see any

**Reply by diriglow** [reply]
Sorry, I saw it on the rpilocator.com site @raychmond mentioned above (thanks btw). I guess they went quick. I’d add the twitter feed for the next time they get em.

**Comment by ChappIO** [comment]
I didn't vote in the poll because I am not US based. But as soon as you'd be willing to send out kits in the EU, I'm in.

**Comment by DoItForDale69** [comment]
Made a github account for the explicit purpose of wanting to buy a PCB. So cool!

**Comment by harlowjg** [comment]
I uploaded the files to OSHPark.
https://oshpark.com/shared_projects/ueCu6qLq

OSHPark purple as one does.  White silkscreen. 

Minimum quantity 3 at $75 USD each.
At quantity 10, price drop to $45 USD.

**Reply by Dp666me** [reply]
I got five for 47 dollars total check other sites

**Reply by stonepreston** [reply]
Oshpark is going to be higher in price (although they are manufactured in the USA if that’s worth it to you). As you noted you only get purple. You can get it cheaper from a China-based company like JLCPCB with more color options, will probably take a little longer.

**Reply by harlowjg** [reply]
Oh, =)  I was not really suggesting the group buy should go this route.  More so wanted to share that the files are in place and processed.

**Reply by stonepreston** [reply]
Oh my bad I missed the link there. That’s cool that you can share it like that!

**Reply by harlowjg** [reply]
Another data point for investigating group buy sources.   
I am not affiliated with or endorsing,  OSHPark or SendCutSend.  (I have seen good results from both services)

Cut Clear Acrylic parts are about $56 USD before shipping.  (other colors are available, some cost more)
(Total case quantity: 1) 
Shared cart from SendCutSend; https://cart.sendcutsend.com/mllepbwyt8v0

Might decease with proper volume buy.... not sure, I have not used them for a large order.
We may be able to sub divide the lower_layer and top_layer in such a way that they nest in a more material efficient manner.   I have intended to lay this out in CAD, but I have not tacked this.

**Reply by Dp666me** [reply]
Thanks when I priced it out it was expensive because each piece needed it's own sheet. If you combine them it would make it cheaper

**Reply by stonepreston** [reply]
If you look in the MCAD repo I show how you can nest the lower and upper layers so you use less material.

**Reply by harlowjg** [reply]
I think we can take it a step further if we can use "L" shaped pieces vs. "G" shapes.

**Reply by stonepreston** [reply]
Id prefer to keep the G shapes so there are no breaks in the walls of the case where the L would come together.

**Comment by MrRampage2321** [comment]
I would absolutely be interested in a group buy for the PCB.

**Comment by evinzant** [comment]
I’m in the US. I have all the parts except the case and the PCB. I looked at PCBWay. I was going to order black with white print. All I’m waiting for is the placement of the PCB order.

**Reply by MrRampage2321** [reply]
Nice. Any interest in selling the extra boards that come with your order?

**Reply by kuffcakes** [reply]
*All three PCB boards are claimed. Will update if a sale falls through.*

I'll have a couple boards for sale when they arrive sometime in the beginning of July. $6 each plus shipping.

**Comment by kuffcakes** [comment]
*All three PCB boards are claimed. Will update if a sale falls through.*

I will have two PCB boards available for sale sometime at the beginning of July. They will be $6 each plus shipping of your choice. I am located in the U.S. and will ship within the U.S.

**Comment by stonepreston** [comment]
For those of you wanting the US group buy, Id prefer to wait another week or 2 before ordering any PCBs as more and more people are joining each day. However I understand wanting to get started so feel free to order if youd like.

**Comment by evinzant** [comment]
I’ll send you six bucks now (PayPal? Apple?) and the shipping when you ship them?

**Comment by rwl4** [comment]
I'd love to get in on this too, along with the acrylic case.

**Comment by darkglove** [comment]
Would like to get this in the UK

**Reply by pixiesue** [reply]
https://github.com/orgs/FundamentalFrequency/discussions/6#discussioncomment-3033054

**Comment by dotdashdotdots** [comment]
Alright! I'm down for a group buy, and since this got featured on a bunch of stuff already, I think we'll easily have the demand... looking forward to lemony goodness!

**Reply by DoItForDale69** [reply]
Sweet! please ping me when you do! would love to get in

**Comment by metaquanta** [comment]
I would like to get on the US group by list! Either option is fine.

**Comment by sounkwn** [comment]
as a musician I would def. like to support this project. Is there anyway I can buy a full build from you? As I do not know how to code or solder.

**Comment by louisnichols** [comment]
Apologies, I missed the detail that this is for people in the US. Was too excited. Hope it won't cause any trouble.

**Comment by dylannorthrup** [comment]
I'm actually thinking about using the files and cnc'ing a wood case, but would definitely be up for getting multiple PCBs and at least one set of the case parts (which are good for comparison to what I manufacture).

**Comment by t-schreibs** [comment]
This project is so cool! 🥺 I would absolutely jump in a group buy for PCB + case if/when it happens.

**Comment by steveb1475** [comment]
I'm interested if still available - US

**Comment by cloudkicker24x** [comment]
Also interested live in the US like another user posted I'd be super interested in paying someone to assemble a working version and ship it to me lol lmk if anyone would be down cuz i have no idea how to code or solder

**Reply by ttdmayshark** [reply]
Yep, ditto, I could do this all myself but I would pay a premium for an assembled unit because time etc

**Reply by mingusdude** [reply]
Hey I should be done with my first one in the next week or so; and if it goes well, I will probably be down to assemble a couple. Probably just parts and a small fee for my time and of course shipping.

**Reply by sounkwn** [reply]
> Hey I should be done with my first one in the next week or so; and if it goes well, I will probably be down to assemble a couple. Probably just parts and a small fee for my time and of course shipping. 

This is awesome! Let me know Id like to get one made and pay you a fee for your time.

**Reply by LucidSD** [reply]
Asked this question earlier but will tease seeing this is a more recent comment thread and related to the US

Sorry to add a redundant question to this board but provided I paid for parts+labor would anyone be able to assist in soldering in Los Angeles, CA. 

This is a project that seems really amazing but my last solder project was just a mess (cheap Ebay solder kit so my partially my fault for getting cheap tools I guess + inexperience) and I ruined some cool retro gear. 

Would be willing to pay for labor to avoid having to buy a whole new soldering kit and risk any bridging mistakes (I read  this is a somewhat tricky solder job due to how close the solder points are to each other)

**Reply by mingusdude** [reply]
Hey sorry on the huge delay here fellas. My extra boards finally arrived, and I would be willing to discuss assembly.

**Reply by LucidSD** [reply]
Sorry I replied on a different post but meant to reply to this one. Would love to discuss assembly

**Reply by LucidSD** [reply]
Do you have an email that’s appropriate? You can reach me at passiverefraction@gmail.com

**Reply by mingusdude** [reply]
So I realize I replied down below, but I wanted to make sure anyone who needs them sees it...
michaelhurtwins@gmail.com

**Comment by mingusdude** [comment]
Just in case anyone is still interested,  I have a couple of extra boards, so just reach out, and we can go from there.

**Reply by harlowjg** [reply]
I am interested in a board set.

**Reply by LucidSD** [reply]
I’m interested in both, though I have a temporary housing crisis situation I need to resolve but I’m definitely interested in this still.

**Reply by interlock** [reply]
Interested in a board as well!

**Reply by mingusdude** [reply]
michaelhurtwins@gmail.com

**Comment by mingusdude** [comment]
I have a coue pcbs left if you are interested.

**Reply by ehafh** [reply]
Do you also have cases?

**Comment by LucidSD** [comment]
Any updates in regards to peoples thoughts on group buying case parts? 
I’m really stuck on the case to be honest, can’t find a cutter in my area (which is weird considering you’d think it would be common here).

Are there any companies that offer services online to cut out a plastic case and just ship it to you? Curious if anyone has looked into this.

**Reply by harlowjg** [reply]
I was going to hold off on this news for a various reasons, but I'll throw it out there as a response to your question.

I've joined my local maker space, I have access to a CNC router and -soon-, I will have access to a laser.   I will probably get clearance to use the laser early next week.   
https://bosslaser.com/boss-ls-3655/

This week I am working on sourcing some acrylic sheets.  I would like to get my hands on a board set, to at least check fitment and such.  Once I get a good case or two, I can start making runs and see if it makes sense to make some for the community.

**Comment by ehafh** [comment]
For the case, is it just a bunch of layers of the same shapes?
If we could get these in board files / gerbers to just print off PCB's it would be really easy for people to make.

**Comment by mingusdude** [comment]
I have no cases unfortunately, and the Gerbers are readily available for
those who don't know, but I also have a couple of boards left if anyone is
interested. ***@***.***

**Comment by inorishizuka** [comment]
I have a small batch of Black and gold plated ( yes the lemons are gold too ) PCB's for the US market if anyone is still looking for some. 

![unnamed2](https://github.com/FundamentalFrequency/.github/assets/17038220/ad25ce52-b347-49d6-8c2f-4875c043fec4)

I also created custom key caps with legends for resin printers:
[https://makerworld.com/en/models/432505](https://makerworld.com/en/models/432505)

![firefox_UrHofMxhhl](https://github.com/FundamentalFrequency/.github/assets/17038220/396b1a94-d233-47a8-83bd-39cb84a51b0d)
![firefox_5ORuvy2c28](https://github.com/FundamentalFrequency/.github/assets/17038220/3bd3bdca-2459-4ce3-a478-91a7fbc6ea95)

I will probably offer 3d Resin printed cases / complete printed key cap sets soon aswell.
